css-fonts-4/Overview.bs

```diff
@@ -2767,6 +2767,40 @@ Font property descriptors: the 'font-style!!descriptor', 'font-weight!!descripto
 		to select a particular face within a font family.
 	</div>
 
+	<div class="example" id="ex-descriptors-range">
+		Fonts may advertise the range of 'font-weight!!property', 'font-stretch!!property', and 'font-style!!property' they are
+		compatible with.
+		<pre>
+		@font-face {
+			font-family: Lastima;
+			src: url(lastima-varfont.woff2);
+			font-weight: 100 399;
+		}
+		</pre>
+		The above ''@font-face'' rule indicates that <code>lastima-varfont.woff</code> should be used when
+		'font-weight!!property' is between 100 and 399. Depending on if there are any other ''@font-face'' rules which specify
+		<code>font-family: Lastima</code>, <code>lastima-varfont.woff</code> might be used for values of 'font-weight!!property'
+		outside of the 100 - 399 range. For more details, see the [[#font-matching-algorithm]].
+
+		As <a href="#ex-descriptors-matching">above</a>, multiple ''@font-face'' rules may be joined together into a single
+		family, spanning multiple ranges of 'font-weight!!property', 'font-stretch!!property', and/or 'font-style!!property':
+		<pre>
+		@font-face {
+			font-family: Lastima;
+			src: url(lastima-varfont-lightrange.woff2);
+			font-weight: 100 399;
+		}
+		@font-face {
+			font-family: Lastima;
+			src: url(lastima-varfont-heavyrange.woff2);
+			font-weight: 400 700;
+		}
+		The above ''@font-face'' rules indicate that <code>lastima-varfont-lightrange.woff</code> should be used when
+		'font-weight!!property' is between 100 and 399, whereas </code>lastima-varfont-heavyrange.woff</code> should be used when
+		'font-weight!!property' is between 400 and 700.
+		</pre>
```

Comment:
@litherum Sorry, late to the party, but this </pre> should probably go after the @font-face rules and before the "The above..." text:

![image](https://user-images.githubusercontent.com/4570664/193749195-dc5de905-6b92-42af-99cb-0e095a79f249.png)

Reply:
https://github.com/w3c/csswg-drafts/commit/9808f1f7d9aa2812bc532220dd590e1452fcc92b